com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Raytracing/LightCluster.cs

```diff
@@ -13,7 +13,7 @@ public sealed class LightCluster : VolumeComponent
         /// Controls the range of the cluster around the camera.
         /// </summary>
         [Tooltip("Controls the range of the cluster around the camera.")]
-        public ClampedFloatParameter cameraClusterRange = new ClampedFloatParameter(10f, 0.001f, 50f);
+        public MinFloatParameter cameraClusterRange = new MinFloatParameter(10.0f, 0.001f);
```

Comment:
Sadly as this is a public value we aren't allowed to rename it... it will be a breaking change. so we will need to change the UX but keep the name

Reply:
ok will update that